RIGS/signals.py

```diff
@@ -102,3 +105,35 @@ def on_revision_commit(sender, instance, created, **kwargs):
 
 
 post_save.connect(on_revision_commit, sender=models.EventAuthorisation)
+
+
+def send_admin_awaiting_approval_email(user, request, **kwargs):
+    # Bit more controlled than just emailing all superusers
+    for admin in models.Profile.objects.filter(email__in=[y for x in settings.ADMINS for y in x]):
+        # Check we've ever emailed them before and if so, if cooldown has passed.
+        if admin.last_emailed is None or admin.last_emailed + datetime.timedelta(minutes=15) <= timezone.now():
+            context = {
+                'request': request,
+                'link_suffix': '/admin/RIGS/profile/?is_approved__exact=0',
```

Comment:
You should be able to write a reverse to the correct admin page?
https://docs.djangoproject.com/en/3.0/ref/contrib/admin/#reversing-admin-urls

Reply:
With the filter applied as well?

Reply:
Good point. It's up to you then, you could reverse the `/admin/RIGS/profile` and the append the query, either way it's fine then!